smart-contracts/contracts/interfaces/IUnlock.sol

```diff
@@ -21,6 +21,11 @@ interface IUnlock {
     string globalTokenURI
   );
 
+  event ResetTrackedValue(
+    uint grossNetworkProduct,
+    uint totalDiscountGranted
+  );
+
```

Comment:
This looks good...but wouldn't `IUnlock.sol` need to have the function signature for `resetTrackedValue` added too?

Reply:
yup, added.